Sample.Api/App/AirportService.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Sample.Api.Model;
+
+namespace Sample.Api.App
+{
+	public class AirportService : HttpClient
+	{
+		public string ApiHost;
+		public AirportService(string apiHost)
+		{
+			ApiHost = apiHost;
+		}
+
+		public List<AirportPair> CalculateDistance(List<Airport> airports)
+		{
+			List<AirportPair> pairs = new List<AirportPair>();
+			Queue<Airport> queue = new Queue<Airport>(airports);
+			Airport currentPort;
+			while (queue.TryDequeue(out currentPort)) {
+				foreach (Airport airport in queue) {
+					if (!currentPort.City.Equals(airport.City)) {
+						var distanceHelper = new DistanceCalculator();
+						pairs.Add(new AirportPair() {
+							First = currentPort,
+							Second = airport,
+							Distance = distanceHelper.DistanceBetweenPlaces(currentPort, airport)
+						});
+					}
+				}
+			}
+
+			return pairs;
+		}
+
+		public async Task<List<Airport>> GetAirports(string[] cities)

```

Comment:
Предлагаю изменить имя метода на
```suggestion
		public async Task<List<Airport>> GetAirportsAsync(string[] cities)

```